en/lessons/shiny-leaflet-newspaper-map-tutorial.md

```diff
@@ -0,0 +1,505 @@
+---
+title: "Making an Interactive Web Application with R and Shiny"
+collection: lessons
+layout: lesson
+slug: shiny-leaflet-newspaper-map-tutorial
+date: 2022-10-19
+authors:
+- Yann Ryan
+reviewers:
+- Amanda Regan
+- Nicole Lemire Garlic
+editors:
+- Alex Wermer-Colan
+review-ticket: https://github.com/programminghistorian/ph-submissions/issues/416
+difficulty: 2
+activity: presenting
+topics: [mapping, website]
+avatar_alt: Reflection of moonlight on a lake
+abstract: This lesson demonstrates how to build an interactive webmap using R and the Shiny library. In the lesson, you will design and implement a simple application, consisting of a slider which allows a user to select a date range, and display a set of corresponding points, on an interactive map.
+lesson-partners: [Jisc, The National Archives]
+partnership-url: /jisc-tna-partnership
+doi: 10.46430/phen0105
+---
+
+{% include toc.html %}
+
+## Introduction
+This lesson demonstrates how to build a basic interactive web application using Shiny. Shiny is a library (a set of additional functions) for the programming language R. Its purpose is to facilitate the development of web applications, which allow a user to interact with R code using User Interface (UI) elements in a web browser, such as sliders, drop-down menus, and so forth. In the lesson, you will design and implement a simple application, consisting of a slider which allows a user to select a date range, which will then trigger some code in R, and display a set of corresponding points on an interactive map.
+
+## Lesson Aims
+
+In this lesson, you will learn:
+
+-   How to create a basic interactive Shiny application.
+-   The key layouts and design principles of the Shiny UI.
+-   The concept and practice of 'reactive programming', as implemented by Shiny applications. Specifically, you'll learn how you can use Shiny to 'listen' for certain inputs, and how they are connected to outputs displayed in your app.
+
+<div class="alert alert-info">
+Note that this lesson doesn't teach any coding in R, other than what's necessary to create the web application, nor does it cover publishing the finished application to the web. A basic knowledge of R, particularly using the [tidyverse](/en/lessons/data_wrangling_and_management_in_R), would be very useful.
```

Comment:
The markdown in this banner isn't rendering 
<img width="756" alt="Screen Shot 2022-10-18 at 10 02 59 PM" src="https://user-images.githubusercontent.com/7232545/196580132-485750aa-187b-4a29-9514-2587decaed58.png">